scripts/generate-static-data.js

```diff
@@ -412,6 +416,8 @@ const texts = [
     'High Barrier',
     'High Barrier ([_1])',
     'Barrier ([_1])',
+    'Reset Barrier',
+    'Reset Barrier ([_1])',
```

Comment:
how about instead of an extra text for `'Reset Barrier ([_1])'`, in js file we only localize Reset Barrier, then append the ` (...)` to the end?

Reply:
sure, should we change this for the Barrier, High Barrier, Low Barrier labels as well?

Reply:
up to you, since they're already translated